src/swap/SwapScreen.tsx

```diff
@@ -107,16 +116,24 @@ export function SwapScreen() {
   }
 
   const handleSelectToken = (tokenAddress: string) => {
+    const selectedToken = coreTokens.find((token) => token.address === tokenAddress)
+    if (selectedToken) {
+      ValoraAnalytics.track(SwapEvents.swap_screen_confirm_token, {
+        fieldType: selectingToken === Field.FROM ? Field.FROM : Field.TO,
```

Comment:
Why not directly: `fieldType: selectingToken`?

Reply:
oh d'oh! thank you for this....